### ISSUE TITLE
mysqueezebox

### ISSUE THREAD

**Issue by kaaholst** [kaaholst/android-squeezer#55]
```
Request from the market: Needs mysqueezebox switch.

```

Original issue reported on code.google.com by `kaaholst` on 2012-02-14 22:51:41

**Comment by kaaholst** [comment]
```
What would this do?  Just switch a player to use mysqueezebox instead of the server?
```

Original issue reported on code.google.com by `nik@ngo.org.uk` on 2012-02-15 22:05:57

**Comment by kaaholst** [comment]
```
Just tried it out and yes... support for mysqueezebox.com as server woudl be very useful.
I exclusively (almost) use MySB.com as server. I cannot remember last I used SBS.
```

Original issue reported on code.google.com by `OppfinnarJocke` on 2012-12-14 20:36:11

**Comment by kaaholst** [comment]
Fixed in 2.0.0

Original issue reported on github.com/nikclayton/android-squeezer by kaaholst on 2020-03-07T19:16:41Z
